ui/apps/platform/cypress/integration/accessControl/accessControlOrigin.test.js

```diff
@@ -0,0 +1,174 @@
+import withAuth from '../../helpers/basicAuth';
+
+import {
+    accessScopesAlias,
+    accessScopesKey,
+    authProvidersAlias,
+    authProvidersKey,
+    permissionSetsKey,
+    groupsAlias,
+    rolesAlias,
+    rolesKey,
+    clickEntityNameInTable,
+    permissionSetsAlias,
+    visitAccessControlEntities,
+} from './accessControl.helpers';
+import { selectors } from './accessControl.selectors';
+
+describe('Access Control declarative Access scopes', () => {
+    withAuth();
+
+    it('declarative access scopes in list view have correct origin and are disabled', () => {
+        const staticResponseMap = {
+            [accessScopesAlias]: {
+                fixture: 'auth/declarativeAccessScope.json',
+            },
+        };
+        visitAccessControlEntities(accessScopesKey, staticResponseMap);
+
+        cy.get('td[data-label="Origin"]').should('have.text', 'Declarative');
+        cy.get('td.pf-c-table__action button[aria-label="Actions"]').should('be.disabled');
+    });
+
+    it('list link for declarative access scope goes to form which has label instead of button and disabled input values', () => {
+        const staticResponseMap = {
+            [accessScopesAlias]: {
+                fixture: 'auth/declarativeAccessScope.json',
+            },
+        };
+        visitAccessControlEntities(accessScopesKey, staticResponseMap);
+
+        const entityName = 'access-scope-test-name';
+        clickEntityNameInTable(accessScopesKey, entityName);
+
+        cy.get(`h2:contains("${entityName}")`);
+        cy.get(`li.pf-c-breadcrumb__item:nth-child(2):contains("${entityName}")`);
+
+        cy.get('.pf-c-label:contains("Declarative")').should('exist');
```

Comment:
In general, `.should('exist')` is redundant because `cy.get(…)` fails if the element does not exist.

Having said that, it is helpful to contrast an explicit exist assertion below with non-exist as a pair, to prevent the subtle potential problem that there is either a mistake or a future change in UI so the not-exist selector cannot possibly exist.